### ISSUE TITLE
failed to build due to chroma dependency

### ISSUE THREAD

**Issue by gh0st42** [dom96/gamelight#5]
I just wanted to prototype some stuff with gamelight and ran into some issues.

gamelight depends on `chroma` being `>= 0.1.0` but 0.2.1 is the most recent version and incompatible to the 0.1 branch, thus, gamelight fails to compile.

It would be nice if you could either pin the dependency to the 0.1 branch or port your code to use the most recent chroma release. Also the same happens in flippy(which is a dependency of gamelight) where the author [suggests](https://github.com/treeform/flippy/issues/35) to use [pixie](https://github.com/treeform/pixie) instead. 
I guess this would be the proper solution.

**Comment by dom96** [comment]
Yeah, sorry, right now I'm very much still prototyping a lot of gamelight. I am using some pretty specific commits of chroma, flippy etc. I'd recommend steering clear until I can make a release of gamelight with the appropriate dependencies.

**Comment by prutschman** [comment]
Would you be willing to specify a working combination of commits, for those of us willing to walk on the wild side?
